### PR TITLE
Tab to space for consistency in parameter mappings.

### PR DESCRIPTION
**Pull request by TDMedina**
Response to solve #268

### CODE REVIEW

File: protocols/CollectGCBiasMetrics.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #string picardJar
 #string insertSizeMetrics
 #string gcBiasMetrics
-#string	project
+^#string project
```
Comment (pneerincx) [inline protocols/CollectGCBiasMetrics.sh:20]: Why is there a ^ at the beginning of the modified lines?

Comment (TDMedina) [reply protocols/CollectGCBiasMetrics.sh:20]: Thaaat's a good question. Looks like I messed up my regex. Maybe just burn this pull request.

Comment (pneerincx) [reply protocols/CollectGCBiasMetrics.sh:20]: Consistent white space is nice though ;)

Comment (TDMedina) [reply protocols/CollectGCBiasMetrics.sh]: Alright, just made it worse, then made it better. should be fixed now.

**Pull request merged**